packaging/nmstate.py3.spec

```diff
@@ -48,6 +48,7 @@ sed -i -e '/^dbus-python$/d' requirements.txt
 %files
 %doc README.md
 %doc examples/
+%{_mandir}/man8/nmstatectl.8.gz
```

Comment:
Please use ``%{_mandir}/man8/nmstatectl.8*`, rpmbuild might compress it to other format in the future.